hrm-domain/hrm-service/migrations/20230926214506-create-client-profiles-tables.js

```diff
@@ -0,0 +1,317 @@
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  up: async queryInterface => {
+    // Profiles
+    await queryInterface.sequelize.query(`
+      CREATE SEQUENCE IF NOT EXISTS public."Profiles_id_seq"
+        INCREMENT 1
+        START 1
+        MINVALUE 1
+        MAXVALUE 9223372036854775807
+        CACHE 1;
+    `);
+    console.log('Sequence "Profiles_id_seq" created');
+    await queryInterface.sequelize.query(`
+    ALTER SEQUENCE public."Profiles_id_seq" OWNER TO hrm
+    `);
+    console.log("Sequence 'Profiles_id_seq' ownership altered.");
+    await queryInterface.sequelize.query(`
+      CREATE TABLE IF NOT EXISTS public."Profiles"
+      (
+        id integer NOT NULL DEFAULT nextval('"Profiles_id_seq"'::regclass),
+        "name" text COLLATE pg_catalog."default",
+        "accountSid" text COLLATE pg_catalog."default" NOT NULL,
+        "createdAt" timestamp with time zone NOT NULL,
+        "updatedAt" timestamp with time zone NOT NULL,
+        CONSTRAINT "Profiles_pkey" PRIMARY KEY ("id", "accountSid")
+      )
+    `);
+    console.log("Table 'Profiles' created.");
+    await queryInterface.sequelize.query(`
+      ALTER TABLE IF EXISTS public."Profiles" OWNER to hrm;
+    `);
+    console.log("Table 'Profiles' ownership altered.");
+
+    // Identifiers
+    await queryInterface.sequelize.query(`
+      CREATE SEQUENCE IF NOT EXISTS public."Identifiers_id_seq"
+        INCREMENT 1
+        START 1
+        MINVALUE 1
+        MAXVALUE 9223372036854775807
+        CACHE 1;
+    `);
+    console.log('Sequence "Identifiers_id_seq" created');
+    await queryInterface.sequelize.query(`
+    ALTER SEQUENCE public."Identifiers_id_seq" OWNER TO hrm
+    `);
+    console.log("Sequence 'Identifiers_id_seq' ownership altered.");
+    await queryInterface.sequelize.query(`
+      CREATE TABLE IF NOT EXISTS public."Identifiers"
+      (
+        id integer NOT NULL DEFAULT nextval('"Identifiers_id_seq"'::regclass),
+        "identifier" text COLLATE pg_catalog."default" NOT NULL,
+        "accountSid" text COLLATE pg_catalog."default" NOT NULL,
+        "createdAt" timestamp with time zone NOT NULL,
+        "updatedAt" timestamp with time zone NOT NULL,
+        CONSTRAINT "Identifiers_pkey" PRIMARY KEY ("id", "accountSid")
+      )
+    `);
+    console.log("Table 'Identifiers' created.");
+    await queryInterface.sequelize.query(`
+      ALTER TABLE IF EXISTS public."Identifiers" OWNER to hrm;
+    `);
+    console.log("Table 'Identifiers' ownership altered.");
+    await queryInterface.sequelize.query(`
+    CREATE INDEX IF NOT EXISTS "Identifiers_identifier_accountSid" ON public."Identifiers" USING btree
+      ("identifier" ASC NULLS LAST, "accountSid" COLLATE pg_catalog."default" ASC NULLS LAST)
```

Comment:
Do we want this table to permit duplicate identifiers for the same account or should we add a unique constraint to this index?

Reply:
Definitely not. good catch! @GPaoloni I think we are replacing this PR/branch with #481 at this point. Could you fix the unique constraint there, please?